packages/govuk-frontend/src/govuk/components/error-summary/_index.scss

```diff
@@ -2,6 +2,7 @@
 
 @include govuk-exports("govuk/component/error-summary") {
   .govuk-error-summary {
+    @include govuk-font($size: 19);
```

Comment:
Same with this one, could it go??

We only need it because we haven't put a heading class on the error summary title?

```html
<h2 class="govuk-error-summary__title"></h2>
```

```suggestion
```

Reply:
This one's tricky. We'd need to apply this to the template, which would be a suggested change which would require a bit of comms. For the sake of getting this PR out as a fix, would you be willing to leave this for now, I make an issue, in 5.X we make this a suggested change and scoop it out wholesale in 6.0?

Reply:
Sounds perfect

E.g. Search for `__title` in `*.njk`

See all the templates where a heading class could make more `govuk-font()` lines unnecessary:

```
.govuk-error-summary__title
.govuk-notification-banner__title
.govuk-panel__title
.govuk-summary-card__title
.govuk-tabs__title
```

Search again with `__heading` and you get the idea 🎉

Definitely one for another issue

Reply:
Making this a wider heading clearup issue is a cracking idea. I'll write up an issue in a bit.